test/libyul/yulSyntaxTests/selfdestruct.yul

```diff
@@ -2,3 +2,4 @@
 	selfdestruct(0x02)
```

Comment:
Looks like we either have no tests using `selfdestruct` in inline assembly or the warning does not show up there. Can you add one? This one tests pure Yul.

Reply:
Are the previous three tests (above) not inline assembly?

Reply:
Haha, they are. Not sure how I missed that :)